components/transactions/TransactionItem.js

```diff
@@ -97,6 +98,18 @@ const KindTag = styled(StyledTag).attrs({
   fontWeight: '600',
 })``;
 
+const getExpenseStatusTag = (status, isRefund, isRefunded) => {
+  let expenseStatusLabel;
+  if (status === expenseStatus.APPROVED && isRefunded) {
+    expenseStatusLabel = expenseStatus.REFUNDED;
+  } else if (status === expenseStatus.APPROVED && isRefund) {
```

Comment:
No reason for me too look at the `expenseStatus` here.

What if the Expense is paid a second-time, rejected after being refunded, etc ... That should not impact the Transaction we're displaying here.

Reply:
Makes sense. I've removed checking for the `expenseStatus`.